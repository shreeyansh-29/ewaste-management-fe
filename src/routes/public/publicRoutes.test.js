/* eslint-disable no-undef */
import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adatper from "@wojtekmaj/enzyme-adapter-react-17";
import toJson from "enzyme-to-json";
import PublicRoutes from "./publicRoutes";

Enzyme.configure({adapter: new Adatper()});

describe("testing authProtected", () => {
  const ResetPass = jest.fn();
  let props;

  it("test", () => {
    const wrapper = shallow(<PublicRoutes {...ResetPass} {...props} />);
    expect.assertions(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
