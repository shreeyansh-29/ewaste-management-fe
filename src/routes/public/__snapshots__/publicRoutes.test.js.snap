// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing authProtected test 1`] = `
<Routes>
  <Route
    element={<HomePage />}
    exact={true}
    path="/"
  />
  <Route
    element={<SignIn />}
    exact={true}
    path="/Signin"
  />
  <Route
    element={<SignUp />}
    exact={true}
    path="/signUp"
  />
  <Route
    element={<Memo(Connect(ForgotPassword)) />}
    exact={true}
    path="/ForgotPassword"
  />
  <Route
    element={<ResetPassword />}
    exact={true}
    path="/password/save/:token"
    render={[Function]}
  />
  <Route
    element={<PageNotFound />}
    exact={true}
    path="*"
  />
</Routes>
`;
